code/modules/jobs/jobs.dm

```diff
@@ -18,6 +18,7 @@ GLOBAL_LIST_INIT(medical_positions, list(
 	"Medical Doctor",
 	"Geneticist",
 	"Virologist",
+	ROLE_EMH, //DeepSpace13 - Emh banning

```

Comment:
odd way to do it. I personally would check for medical doctor and/or ghostrole

Reply:
This way we can ban from EMH but not MD.